kernel/src/main/scala/org/apache/toree/boot/layer/ComponentInitialization.scala

```diff
@@ -116,12 +117,17 @@ trait StandardComponentInitialization extends ComponentInitialization {
   }
 
   private def initializeDependencyDownloader(config: Config) = {
-    /*val dependencyDownloader = new IvyDependencyDownloader(
-      "http://repo1.maven.org/maven2/", config.getString("ivy_local")
-    )*/
+    val depsDir = {
+      if(config.hasPath("deps_dir") && Files.exists(Paths.get(config.getString("deps_dir")))) {
+        config.getString("deps_dir")
+      } else {
+        Files.createTempDirectory("spark_kernel_add_deps").toFile.getAbsolutePath
```

Comment:
maybe we should start using `toree_add_deps` instead of `spark_kernel`. Should change in addjar if necessary.